### PR TITLE
single combined sampleqc path

### PR DESCRIPTION
**Pull request by annacuomo**
Use combined tenk10k sample QC path instead of a TOB + BioHEART

- leaving the option of cohort-specific sample QC HT files commented out for now.
- updating files to use in the instructions at the top to most recent ones
- using sex_karyotype but converted to numeric coding

### CODE REVIEW

File: get_sample_covariates.py [hunk 1]
```diff
@@ -85,35 +79,35 @@ def main(
...
-    # sample_qc_df['sample_id'] = [str(s) for s in sample_qc_df['s']]
-    # # only retain relevant columns
-    # sex_df = sample_qc_df[['sample_id', 'sex']]
+    sample_qc_ht = hl.read_table(tenk10k_sampleqc_file_path)
+    # convert to pandas
+    sample_qc_df = sample_qc_ht.to_pandas()
+    # extract info and reformat to table
+    sample_qc_df['sample_id'] = [str(s) for s in sample_qc_df['s']]
+    # only retain relevant columns
+    sex_df = sample_qc_df[['sample_id', 'sex']]
```
Comment (katiedelange) [inline get_sample_covariates.py]: This column field will need to be 'sex_karyotype' to use the genetically the inferred sex.

**Review by katiedelange — CHANGES_REQUESTED** [review]
The path in the run command at the top is correct, but you will need to adjust the column you point to so that you use inferred sex rather than the (incomplete) reported sex column.

File: get_sample_covariates.py (hunk 1 above)
Comment (annacuomo) [reply get_sample_covariates.py]: Yup I just noticed and made the change!

File: get_sample_covariates.py [hunk 2]
```diff
@@ -85,35 +79,39 @@ def main(
...
-    # # only retain relevant columns
-    # sex_df = sample_qc_df[['sample_id', 'sex']]
+    sample_qc_ht = hl.read_table(tenk10k_sampleqc_file_path)
+    # convert to pandas
+    sample_qc_df = sample_qc_ht.to_pandas()
+    # extract info and reformat to table
+    sample_qc_df['sample_id'] = [str(s) for s in sample_qc_df['s']]
+    # go from sex karyotype (XX, XY) to sex numeric coding (2,1)
+    sample_qc_df['sex'] = (
+        sample_qc_df['sex_karyotype'].map({"XY": 1, "XX": 2, "X": 0}).astype(int)
```
Comment (katiedelange) [inline get_sample_covariates.py:111]: Looks good for now, but let's come back and generalise this to better capture more complex sex aneuploids.

Comment (annacuomo) [reply get_sample_covariates.py:111]: I'll make an issue!

Comment (annacuomo) [reply get_sample_covariates.py:111]: https://github.com/populationgenomics/saige-tenk10k/issues/173

**Pull request merged**